app/models/patient.rb

```diff
@@ -49,7 +49,7 @@ class Patient < ApplicationRecord
     -> { where(identifier_type: "simple_bp_passport").order(device_created_at: :desc) },
     class_name: "PatientBusinessIdentifier"
 
-  has_many :current_prescription_drugs, -> { where(is_deleted: false) }, class_name: "PrescriptionDrug"
+  has_many :current_prescription_drugs, -> { where(is_deleted: false).order(created_at: :desc) }, class_name: "PrescriptionDrug"
```

Comment:
Something about the field additions caused the ordering to become indeterministic.

Reply:
Why does this association need to be ordered? I recall that ordered associations have caused us problems when we compose queries together in ActiveRecord.

Reply:
The current_prescription_drugs were being ordered weirdly (not by date or name) by Postgres, hence this to have some order (a spec broke).

> I recall that ordered associations have caused us problems when we compose queries together in ActiveRecord.
The `in_batches` in the exporter removes `order` scopes, however we call this method separately on a `patient` so it works for us.